FilterComplexCore.js

```diff
@@ -338,15 +370,17 @@ module.exports = class FilterComplexCore extends ABComponent {
             if (!Array.isArray(value)) value = [value];
 
             result =
-               (value || []).filter((v) => (v.id || v) == this.Account.username)
-                  .length > 0;
+               (value || []).filter(
```

Comment:
`value ?? []`

Reply:
As I look closer at this code shouldn't it just be `value.filter(` because the code at line 370 will ensure it is an array already.

Reply:
Ok, Please remove `|| []`. Thank you.

Reply:
Ohh... Good catch.

If it is `Array.isArray(null)`, then we will have `[null]`.

Reply:
Ok, Please remove `|| []`, thank you.